packages/aa-comunica-parser/lib/ComunicaSparqlParser.ts

```diff
@@ -0,0 +1,87 @@
+import type { RuleDefReturn } from '@traqula/core';
+import { createToken, GeneratorBuilder, LexerBuilder, ParserBuilder } from '@traqula/core';
+import { sparql12GeneratorBuilder } from '@traqula/generator-sparql-1-2';
+import { sparql12ParserBuilder } from '@traqula/parser-sparql-1-2';
+import { gram as g11 } from '@traqula/rules-sparql-1-1';
+import type * as T11 from '@traqula/rules-sparql-1-1';
+import { lex as l12, completeParseContext } from '@traqula/rules-sparql-1-2';
+import type { SparqlQuery } from '@traqula/rules-sparql-1-2';
+import type * as T12 from '@traqula/rules-sparql-1-2';
+import type { PatternLateral } from './ComunicaSparqlAst';
+
+// eslint-disable-next-line require-unicode-regexp
+const lateral = createToken({ name: 'Lateral', pattern: /lateral/i, label: 'Lateral pattern' });
+const lateralLexer = LexerBuilder
+  .create(l12.sparql12LexerBuilder)
+  .add(lateral);
+
+const graphPatternNotTriples: T12.SparqlRule<
+  (typeof g11.graphPatternNotTriples)['name'],
```

Comment:
Within the comunica codebase (and beyond), we tend to avoid cryptic variable names like `g11`, `$`, `gImpl`, `impl`, `origSource`, and so on.
Writing words in full improves readability IMO.
What is your opinion on this?